src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.cs

```diff
@@ -25,20 +25,19 @@ public static partial class JsonSerializer
             {
                 do
                 {
-                    WriteStackFrame current = state.Current;
```

Comment:
This caused a copy of the struct. It was a regression where I believe it was unintentionally changed for readability.

Reply:
Is there a test case that would expose the struct copy resulting in a bug? Please add one.

Reply:
This bug didn't have any side affects other than perf -- the `state.Current` is not modified directly by any other the called methods.